bitrise.yml

```diff
@@ -687,9 +695,13 @@ workflows:
     - _install-and-set-env
     - _pull_app_files
     after_run:
-    - _run_android_tests
+    - _run_android_full_regression
```

Comment:
isn't this going to run regression each time?  what happens when we want to run smoke tests?